src/portal/azext_portal/aaz/latest/portal/dashboard/_create.py

```diff
@@ -49,7 +49,6 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="The name of the dashboard.",
             required=True,
             fmt=AAZStrArgFormat(
-                pattern="^[a-zA-Z0-9-]{3,24}$",
```

Comment:
why not change it to {3,64}

Reply:
There is no pattern before bumping api version, and actually a dashboard could have a name longer then 64 characters, I think let the server check the name length?